### PR TITLE
marqeta-authorization-byTokenId: get authoriztaion by token id

### PR DESCRIPTION
**Pull request by jbearden**

### CODE REVIEW

File: src/authorization-control.ts [hunk 1]
```diff
@@ -65,4 +65,39 @@ export class AuthorizationControlApi {
...
+    token?: string,
+    fields?: string[],
+  } = {}): Promise<{
+    success: boolean,
+    body?: AuthorizationControl,
+    error?: MarqetaError,
+  }> {
+    const {
+      token,
+      fields = ''
```
Comment (drbobbeaty) [inline src/authorization-control.ts:84]: If `fields` is a `string[]`... then how is it defaulted to the empty `string`? And given that `fire()` will remove empty strings, why not just say:

```typescript
const {
  token,
  fields,
} = search
```

??

**Review by drbobbeaty — CHANGES_REQUESTED** [review]
a question of types and defaults

File: src/authorization-control.ts (hunk 1 above)
Comment (jbearden) [reply src/authorization-control.ts:84]: ` Fire()` does not allow for `'string[] | undefined'`, therefore, compiler complains:
 
` <html>TS2322: Type 'string[] | undefined' is not assignable to type 'string | number | boolean | string[]'.<br/>Type 'undefined' is not assignable to type 'string | number | boolean | string[]'.`

Comment (jbearden) [reply src/authorization-control.ts:84]: nvm, got it...

**Review by drbobbeaty — APPROVED** [review]

**Pull request merged**